jerry-debugger/jerry-client-ws.py

```diff
@@ -81,6 +81,10 @@ def arguments_parse():
                         help="disable stop when newline is pressed (default: %(default)s)")
     parser.add_argument("--color", action="store_true", default=False,
                         help="enable color highlighting on source commands (default: %(default)s)")
+    parser.add_argument("--display", action="store", default="",
```

Comment:
The default should be `None`. Also you can specify the type for which the value will be converted automatically (just add the `type=int` as a keyword argument after the `default` entry.